config/jobs/kubeflow/OWNERS

```diff
@@ -0,0 +1,2 @@
+approvers:
+- jlewi
```

Comment:
@jlewi this is the OWNERS file, feel free to change it and add more people here.

Reply:
Thanks!
/lgtm